src/t8_schemes/t8_standalone/t8_standalone_implementation.hxx

```diff
@@ -0,0 +1,1568 @@
+
+    for (size_t coord = 0; coord < num_coords; ++coord) {
+      for (int dim = 0; dim < T8_ELEMENT_DIM[TEclass]; ++dim) {
+        current_out_coords[dim] = ((t8_standalone_element<TEclass> *) elem)->coords[dim] + current_ref_coords[dim] * h;
+
+        current_out_coords[dim] /= (double) t8_standalone_scheme<TEclass>::get_root_len ();
+      }
+
+      current_ref_coords += T8_ECLASS_MAX_DIM;
+      current_out_coords += T8_ELEMENT_DIM[TEclass];
+    }
+  }
+
+  // ################################################____MEMORY____################################################
+
+  /** Allocate memory for an array of elements of a given class and initialize them.
+   * \param [in] length   The number of elements to be allocated.
+   * \param [in,out] elems On input an array of \b length many unallocated
+   *                      element pointers.
+   *                      On output all these pointers will point to an allocated
+   *                      and initialized element.
+   * \note Not every element that is created in t8code will be created by a call
+   * to this function. However, if an element is not created using \ref t8_element_new,
+   * then it is guaranteed that \ref t8_element_init is called on it.
+   * \note In debugging mode, an element that was created with \ref t8_element_new
+   * must pass \ref t8_element_is_valid.
+   * \note If an element was created by \ref t8_element_new then \ref t8_element_init
+   * may not be called for it. Thus, \ref t8_element_new should initialize an element
+   * in the same way as a call to \ref t8_element_init would.
+   * \see t8_element_init
+   * \see t8_element_is_valid
+   */
+  /* TODO: would it be better to directly allocate an array of elements,
+   *       not element pointers? */
+  void
+  element_new (int length, t8_element_t **elem) const
+  {
+    /* allocate memory */
+    T8_ASSERT (this->scheme_context != NULL);
+    T8_ASSERT (0 <= length);
+    T8_ASSERT (elem != NULL);
+
+    for (int i = 0; i < length; ++i) {
+      elem[i] = (t8_element_t *) sc_mempool_alloc ((sc_mempool_t *) this->scheme_context);
+    }
+
+/* in debug mode, set sensible default values. */
+#ifdef T8_ENABLE_DEBUG
+    {
+      int i;
+      for (i = 0; i < length; i++) {
+        t8_standalone_scheme<TEclass>::element_init (1, elem[i]);
+      }
+    }
+#endif
+  }
+
+  /** Initialize an array of allocated elements.
+   * \param [in] length   The number of elements to be initialized.
+   * \param [in,out] elems On input an array of \b length many allocated
+   *                       elements.
+   * \param [in] called_new True if the elements in \a elem were created by a call
+   *                       to \ref t8_element_new. False if no element in \a elem
+   *                       was created in this way. The case that only some elements
+   *                       were created by \ref t8_element_new should never occur.
```

Comment:
```suggestion
```

Reply:
please update this comment in general.

Reply:
`called_new` is no longer part of the interface, but the comment was not updated in all of the default files as well and should be updated here

Reply:
In the documentation you still reference to called_new, Please update the documentation.